libvcx/src/libindy/proofs/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod prover;
+pub mod verifier;
+pub mod proof_request;
+mod proof_request_internal;
```

Comment:
Good idea separating public / private data structures 👍

Reply:
yup, there was a whole bunch of functions which were basically only consumed by either `validate_indy_proof` or `generate_indy_proof`

Reply:
I would like to see us separating these more often.